code/modules/jobs/job_types/agent.dm

```diff
@@ -61,6 +61,10 @@
 			ears = /obj/item/radio/headset/headset_records
 			accessory = /obj/item/clothing/accessory/armband/lobotomy/records
 
+		else	//Pick a department or get training.
```

Comment:
If that's the case, remove the Training from the switch itself.

Reply:
Good point; i overlooked that.